Samples/AppLifecycle/Activation/cs-console-unpackaged/CsConsoleActivation/CsConsoleActivation.csproj

```diff
@@ -3,18 +3,12 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
-    <Platforms>x64;x86</Platforms>
+    <Platforms>x64;x86;ARM64</Platforms>
     <RuntimeIdentifiers>win10-x64;win10-x86</RuntimeIdentifiers>
```

Comment:
Also win10-arm64 in the RuntimeIdentifiers?

Reply:
Fixed.